.github/workflows/test.yml

```diff
@@ -129,7 +129,10 @@ jobs:
         run: make check-markdown
       - name: Check Component Docs
         if: needs.changes.outputs.source == 'true' || needs.changes.outputs.component_docs == 'true'
-        run: make check-component-docs
+        uses: nick-fields/retry@v2
+        with:
+          max_attempts: 3
+          command: make check-component-docs
```

Comment:
Doesn't it require a timeout setting?

Reply:
Mmm, you are right, I thought there was a default. I'll add.